### DISCUSSION TITLE
Saving ASReview outcomes in other format than .h5

### DISCUSSION THREAD

**Discussion by MartijnOW** [asreview/asreview#1216] [Q&A] [answered]
Hi ASReview community,

During my exploration of ASReview, I came across a few points that were not entirely clear to me. So, I would like to ask those in three separated discussions and I hope you can help me out! Just to explain the context: I want to perform a batch simulation (from the command prompt).

Is the only output file possible the .h5 file? I noticed that this file uses a lot of memory (because it probably stores a lot of information), whereas a table existing of two columns consisting of the row number of articles and the location in the ranking would be sufficient for me to perform some analyses on efficiency of the tool. 

Also, I found out that the .h5 file isn’t very easy to convert to other file types (again, probably due to all information that is stored) and thus I have manually copied and pasted all results (by opening them one by one in the command prompt to an excel sheet) for an initial analysis. However, I’m sure that this could be done much more efficiently and I hope you can help me out!

**Comment by Rensvandeschoot** [comment, marked as answer]
Indeed, the `h5` files can become quite large! Therefore, we introduced a new data model in version 1.0, see https://github.com/asreview/asreview/pull/694. The new project file (stored as `.asreview`) is much smaller. Also, its contents can be accessed via the [ASReview Python API](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html).

**Reply by MartijnOW** [reply]
Thank you, that is great! Just a quick follow-up question: When I look into the file what has been changed in version 1.0, I see the following: "* Refactor simulate entry point and deprecate simulate-batch (#835)". And I indeed get an error when I try to use my "old" command with 'simulate-batch XXX --state_file XXXX.asreview --n_run 10'. 

Is it still possible to run multiple simulations in one command in ASReview version 1.0? I cannot find anything about this in the documentation.

**Reply by Rensvandeschoot** [reply]
even better! see the new tool [Makita](https://github.com/asreview/asreview-makita) which can create batch scripts for large simulation studies.
